### PR TITLE
Add jitpack instructions to android page

### PR DESCRIPTION
**Pull request by tumarsal**
Add instruction to install sdk-cubic cross jitpack

### CODE REVIEW

File: docs-src/content/using-cubic-sdk/installation.md [hunk 1]
```diff
@@ -34,8 +34,27 @@ dotnet add package Cubic-SDK
...
+Add it in your root build.gradle at the end of repositories:
+``` bash
+allprojects {
+		repositories {
+			...
+			maven { url 'https://jitpack.io' }
+		}
+	}
+```
+Add dependency:
```
Comment (renaissanceGeek) [inline docs-src/content/using-cubic-sdk/installation.md:49]:
```suggestion
Then add sdk-cubic as a dependency:
```

File: docs-src/content/using-cubic-sdk/installation.md [hunk 2]
```diff
@@ -34,8 +34,27 @@ dotnet add package Cubic-SDK
 ```
 
 ### Android
-Building for Android requires more steps, so it is described on the [Android Integrations](../android/) page.
 
+##### JitPack 
+
+Add it in your root build.gradle at the end of repositories:
```
Comment (renaissanceGeek) [inline docs-src/content/using-cubic-sdk/installation.md:40]:
```suggestion
#### Using Gradle and JitPack 

To use the pre-built SDK libraries, add JitPack in your root build.gradle at the end of `repositories`:
```

File: docs-src/content/using-cubic-sdk/installation.md [hunk 3]
```diff
@@ -34,8 +34,27 @@ dotnet add package Cubic-SDK
...
+```
+Add dependency:
+``` bash
+dependencies {
+    implementation 'com.github.cobaltspeech:sdk-cubic:v1.6.0'
+}
+```
+##### From Souce code
+
+Building for Android requires more steps, so it is described on the [Android Integrations](../android/) page.
```
Comment (renaissanceGeek) [inline docs-src/content/using-cubic-sdk/installation.md:57]:
```suggestion
#### From Source Code

Building for Android from source requires more steps, so it is described on the [Android Integrations](../android/) page.
```

File: docs-src/content/using-cubic-sdk/installation.md [hunk 4]
```diff
@@ -34,8 +34,27 @@ dotnet add package Cubic-SDK
 ```
 
 ### Android
-Building for Android requires more steps, so it is described on the [Android Integrations](../android/) page.
 
+##### JitPack 
+
+Add it in your root build.gradle at the end of repositories:
+``` bash
```
Comment (renaissanceGeek) [inline docs-src/content/using-cubic-sdk/installation.md:41]:
```suggestion
``` json
```
Also, fix the indentation

File: docs-src/content/using-cubic-sdk/installation.md [hunk 5]
```diff
@@ -34,8 +34,27 @@ dotnet add package Cubic-SDK
...
+allprojects {
+		repositories {
+			...
+			maven { url 'https://jitpack.io' }
+		}
+	}
+```
+Add dependency:
+``` bash
```
Comment (renaissanceGeek) [inline docs-src/content/using-cubic-sdk/installation.md:50]:
```suggestion
``` json
```

**Review by renaissanceGeek — APPROVED** [review]
That will make life easier for our Android customers, thanks!  I made some suggestions, but if you accept the suggestions, I think it's good to merge.

File: docs-src/content/using-cubic-sdk/installation.md [hunk 6]
```diff
@@ -58,10 +58,31 @@ The Cubic SDK is published to the jitpack repository.  To build using Maven, add
...
+	repositories {
+		...
+		maven { url 'https://jitpack.io' }
+	}
+}
+```
+Then add sdk-cubic as a dependency:
+``` gradle
+dependencies {
+    implementation 'com.github.cobaltspeech:sdk-cubic:v1.6.0'
```
Comment (jhollowayj) [inline docs-src/content/using-cubic-sdk/installation.md:81]: Should we add this to the version bumping script?  Or we could say something like 1+, but even that could go stale.

File: docs-src/content/using-cubic-sdk/installation.md [hunk 7]
```diff
@@ -58,10 +58,31 @@ The Cubic SDK is published to the jitpack repository.  To build using Maven, add
 </dependencies>
 ```
 3. You should then be able to build your project with  `mvn package`
+
 #### Android
 #### From Source code
```
Comment (jhollowayj) [inline docs-src/content/using-cubic-sdk/installation.md:62]: This was duplicated down below, so these three lines can be deleted I think.

**Review by jhollowayj — COMMENTED** [review]
Almost there.

File: docs-src/content/using-cubic-sdk/installation.md (hunk 6 above)
Comment (tumarsal) [reply docs-src/content/using-cubic-sdk/installation.md:76]: I add replace version scripts to readme

**Review by kapitoshka438 — APPROVED** [review]
Me gusta

**Pull request merged**